siege_utilities/geo/grids.py

```
@@ -0,0 +1,163 @@
+    Returns:
+        ``pd.Series`` of cell IDs (H3 hex strings or S2 cell tokens by
+        default).
+    """
+    chosen = infer_grid(grid, {"resolution": resolution, "level": level})
+    if chosen == "h3":
+        from .h3_utils import h3_index_points
+        if level is not None and resolution is None:
+            # Caller asked for grid='h3' but used 'level'; honour it.
+            resolution = level
+        return h3_index_points(df, lat_col, lon_col, resolution or 8)
+    else:
```

Ruff (0.15.12)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)